lib/fprotect/Kconfig

```diff
@@ -23,7 +23,7 @@ endif
 config FPROTECT_BLOCK_SIZE
 	hex
 	default NRF_SPU_FLASH_REGION_SIZE if SOC_NRF9160 || SOC_NRF5340_CPUAPP
-	default $(dt_hex_val,DT_SOC_NV_FLASH_1000000_ERASE_BLOCK_SIZE) if SOC_NRF5340_CPUNET
-	default $(dt_hex_val,DT_SOC_NV_FLASH_0_ERASE_BLOCK_SIZE)
+	default $(dt_node_int_prop_hex,/flash-controller@41080000/flash@1000000,erase-block-size) if SOC_NRF5340_CPUNET
```

Comment:
@mbolivar-nordic Iooks like this is proper fix (works well in the build)

Reply:
If there is no other option, OK. But this is a bit hard-coded, especially for the default value in the next line. I will try to propose a different solution.

Reply:
Ok, if you take this pull request in zephyr as `[nrf fromlist]` patches: https://github.com/zephyrproject-rtos/zephyr/pull/24654

Then you should be able to do this instead:

```
diff --git a/lib/fprotect/Kconfig b/lib/fprotect/Kconfig
index 93a2c196ea..6749fabfc6 100644
--- a/lib/fprotect/Kconfig
+++ b/lib/fprotect/Kconfig
@@ -20,10 +20,13 @@ config NRF_SPU_FLASH_REGION_SIZE
          Redefinition for the benefit of qemu_x86
 endif
 
+# Workaround for not being able to have commas in macro arguments
+DT_ZEPHYR_FLASH := zephyr,flash
+DT_CHOSEN_ZEPHYR_FLASH := $(dt_chosen_path,$(DT_ZEPHYR_FLASH))
+
 config FPROTECT_BLOCK_SIZE
        hex
        default NRF_SPU_FLASH_REGION_SIZE if SOC_NRF9160 || SOC_NRF5340_CPUAPP
-       default $(dt_node_int_prop_hex,/flash-controller@41080000/flash@1000000,erase-block-size) if SOC_NRF5340_CPUNET
-       default $(dt_node_int_prop_hex,/soc/flash-controller@4001e000/flash@0,erase-block-size) if SOC_SERIES_NRF52X || SOC_SERIES_NRF51X
+       default $(dt_node_int_prop_hex,$(DT_CHOSEN_ZEPHYR_FLASH),erase-block-size)
        help
          Block size of fprotect
```

Reply:
will try out that.

Reply:
> will try out that.

I went ahead and pushed it.